### ISSUE TITLE
What is the experimental setup for human mesh recovery?

### ISSUE THREAD

**Issue by HospitableHost** [cure-lab/SmoothNet#49]
This table is from your paper.
![image](https://github.com/cure-lab/SmoothNet/assets/44576391/80fe8f21-5b41-46e4-a772-a74774e6ab40)
In the paper, you mentioned that "SmoothNet is trained with the pose outputs from SPIN [22]. We test its performance across multiple backbone networks."
But you didn't describe the experimental configuration. So which dataset are you using poses from? And what size is your sliding window here?
Also, did you evaluate the metrics using the test set of the three datasets? Because I run your eval code, but the results of vibe is different from this table.
![image](https://github.com/cure-lab/SmoothNet/assets/44576391/1dc198d4-1ebe-4bdc-82e4-856c1b656c63)
(data from pw3d_vibe_smpl_test.npz and pw3d_gt_smpl_test.npz)(and the vibe results are not affected by smoothnet pre-trained models)

**Comment by HospitableHost** [comment]
In addition, I noticed that the data from data/poses/pw3d_vibe_smpl contains 37 sequences. But this does not match with the 3DPW dataset, which has 24 sequences in test_set, 12 sequences in vald_set and 24 sequences in train_set.
![image](https://github.com/cure-lab/SmoothNet/assets/44576391/349a11aa-d4e0-45fd-b5f0-aaf97f6f0184)

**Comment by ailingzengzzz** [comment]
Hi @HospitableHost ,

We used the 3dpw-spin for training. The differences may come from that we removed a few frames at the end of each period that cannot be divided by sliding windows. Did you use our provided code and data for the test?

For 37 sequences, there are some sequences that have two persons. We simply split them into two sequences.

**Comment by HospitableHost** [comment]
@ailingzengzzz Hi, I read your code carefully and tested with your code and data. (data/poses/pw3d_vibe_smpl/...) Therefore, I don't know what caused the VIBE results to be different from your paper.
![image](https://github.com/cure-lab/SmoothNet/assets/44576391/8df72f74-23d7-43de-bbb2-90b5a986bbf6)
![image](https://github.com/cure-lab/SmoothNet/assets/44576391/a1500028-c891-43f2-bca0-87ee9268fcf1)


By the way, I still have three questions:
first, did you use spin---3dpw_train_set for training, right?
second, do the 37 sequences correspond to the 3dpw_test_set?
third, could you please open-source the pretrained model: 3DPW-SPIN-SMPL (not 3DPW-SPIN-3D)

best wishes!

**Comment by ailingzengzzz** [comment]
Hi @HospitableHost,

The answer to questions 1 and 2 is yes. 
We explored the 3DPW-SPIN-SMPL for SMPL (6d rotation matrix) testing but found it is inferior to 3DPW-SPIN-3D (see the paper). Thus, we only provided the 3d model for cross-modality testing.

**Comment by HospitableHost** [comment]
Hi @ailingzengzzz 
So have you figured out why the results of VIBE are inconsistent with the paper?
I suspect that there is something wrong with the data in the folder data/poses/pw3d_vibe_smpl.

**Comment by HospitableHost** [comment]
Besides, I run the evalution of this command :
        CUDA_VISIBLE_DEVICES=7 python eval_smoothnet.py --cfg configs/pw3d_spin_3D.yaml --checkpoint data/checkpoints/pw3d_spin_3D/checkpoint_32.pth.tar --dataset_name pw3d --estimator spin --body_representation smpl --slide_window_size 32

The result is also different from your paper.
![image](https://github.com/cure-lab/SmoothNet/assets/44576391/5da08aea-914b-4793-8fa6-d5c9de5a85ac)
![image](https://github.com/cure-lab/SmoothNet/assets/44576391/34bdbf6f-a8d8-4086-816f-d59deb3ef8ff)

**Comment by ailingzengzzz** [comment]
Hi @HospitableHost ,

The results in Table 3 are calculated via 3D keypoint positions following previous works (e.g., VIBE, PARE, HMR etc). They use a model to estimate SMPL parameters and transform them into 3D keypoint positions for MPJPE, PA-MPJPE, and ACCEL. We simply input the transformed 3D keypoint into SmoothNet to obtain the final 3D keypoint positions.

**Comment by donghaoye** [comment]
> Besides, I run the evalution of this command : CUDA_VISIBLE_DEVICES=7 python eval_smoothnet.py --cfg configs/pw3d_spin_3D.yaml --checkpoint data/checkpoints/pw3d_spin_3D/checkpoint_32.pth.tar --dataset_name pw3d --estimator spin --body_representation smpl --slide_window_size 32
> 
> The result is also different from your paper. ![image](https://private-user-images.githubusercontent.com/44576391/249377765-5da08aea-914b-4793-8fa6-d5c9de5a85ac.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk2ODA2NzMsIm5iZiI6MTcyOTY4MDM3MywicGF0aCI6Ii80NDU3NjM5MS8yNDkzNzc3NjUtNWRhMDhhZWEtOTE0Yi00NzkzLThmYTYtZDVjOWRlNWE4NWFjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMjMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDIzVDEwNDYxM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFiNmFkN2UyNzA3Y2ZiOGRlMDA2Y2EyY2Q2MjY2OGU3YTQyY2MxMDU2YzZhYzAzMGQ3MzZlMGNlZmE3MDE3NzEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.VNXtN6KEXPJ4akZtaZcDIc39S1NhsmM3rCrExkscTPI) ![image](https://private-user-images.githubusercontent.com/44576391/249377798-34bdbf6f-a8d8-4086-816f-d59deb3ef8ff.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk2ODA2NzMsIm5iZiI6MTcyOTY4MDM3MywicGF0aCI6Ii80NDU3NjM5MS8yNDkzNzc3OTgtMzRiZGJmNmYtYThkOC00MDg2LTgxNmYtZDU5ZGViM2VmOGZmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMjMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDIzVDEwNDYxM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTBiNDE0OTM4M2FhNDUyNmIyOWZmZGU1MjFkZmE4ZTk1YjliNzg0MDcwMjMyNzNlYTE1ZTdjMzZhMjA4ODYxNzgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.BAZs_YGJCmhzf20rcdSSzsZvS1Mva2oarD9mPj0s7Cw)

@HospitableHost 
Hi, Have you fixed the above problem?

**Comment by HospitableHost** [comment]
@donghaoye 
Hi, you could refer to my paper "MoManifold: Learning to Measure 3D Human Motion via Decoupled Joint Acceleration Manifolds" for some details of my fair comparison to SmoothNet. 
By the way, if you have some questions, you could send me email or add my wechat. 
In fact, their results are different from the paper indeed and some data utilized in their paper somehow lost in their server.
So I just use the released code to report evalution results and I add the footnote in my paper.
